DependencyInjection/Configuration.php

```diff
@@ -269,6 +270,10 @@ private function addViewSection(ArrayNodeDefinition $rootNode)
 
     private function addBodyListenerSection(ArrayNodeDefinition $rootNode)
     {
+        $decodersDefaultValue = ['json' => 'fos_rest.decoder.json'];
+        if (class_exists(XmlDecoder::class)) {
```

Comment:
What's the point of this? This class always exists. You need to check against `Symfony\Component\Serializer\Encoder\XmlEncoder` ...

Reply:
wow I'm really sorry about that. I should focus more when doing a pull request.